### DISCUSSION TITLE
Generate app.html

### DISCUSSION THREAD

**Discussion by MentalGear** [sveltejs/kit#9634] [Ideas]
I think there should be a way to (re-)generate the app.html automatically. I have an app.html which I have taken over from an old project, and I'm not sure if it is still the right configuration. There's no template on the website, and seemingly no way to generate a up to date app.html as it should be.

**Comment by aloker** [comment]
You'll find the current template at https://github.com/sveltejs/kit/blob/master/packages/create-svelte/templates/skeleton/src/app.html

Maybe that helps (in combination with a little curl invocation)
